vllm/engine/arg_utils.py

```diff
@@ -165,6 +165,7 @@ class EngineArgs:
     num_scheduler_steps: int = 1
     multi_step_stream_outputs: bool = True
     ray_workers_use_nsight: bool = False
+    ray_placement_group: Optional[Any] = None
```

Comment:
is it possible to automatically detect the current placement group the process belongs to, and pass it directly?

ray_placement_group is not really an engine arg, i would say.

Reply:
No it's not possible.

Reply:
which one is not possible?

Reply:
@comaniac we can detect the pg before the fork happens right?

Reply:
Oh I see. Yeah it works but we just need to import ray in `create_engine_config`